pkg/agent/clean/orphan_bindings.go

```diff
@@ -157,13 +166,16 @@ func (bc *orphanBindingsCleanup) cleanOrphans() error {
 	return returnErr
 }
 
-// isOrphanBinding detects whether or not rb is an orphaned binding. Only bindings with a Group subject may be orphans.
+// isOrphanBinding detects whether a role binding is orphaned. Only bindings with a Group subject may be orphans.
 // To detect orphans, we look at labels with value == PrtbInClusterBindingOwner. If the key for this label is a UID,
 // then it is considered to be legacy as the format for the key on this label was changed with 2.5. If the label's key is of the
 // form <ns>_<prtb-name> then label is considered to be new (post 2.5.0). If we find a binding with a legacy label and not a new label, we check if there is an
 // existing prtb with that uid. If there is not, then the binding is an orphan. If a new label exists on the binding, then we check if the parent
 // prtb exists. If it does not exist, the binding is an orphan.
-func (bc *orphanBindingsCleanup) isOrphanBinding(rb rbacv1.RoleBinding) (bool, error) {
+func (bc *orphanBindingsCleanup) isOrphanBinding(rb *rbacv1.RoleBinding) (bool, error) {
```

Comment:
Nit: It occurs to me that this function no longer ever returns an error, so it would probably be good to remove the error return.